cards/base.py

```diff
@@ -15,6 +15,18 @@ class Card:
     adhocturn_action_title = ''
     adhocturn_form = None
 
+    def __eq__(self, other):
```

Comment:
These two guys are needed in order to get sorting of instances and allow making `Card` objects the keys in dicts.